components/omega/doc/design/OmegaV1GoverningEqns.md

```diff
@@ -0,0 +1,1589 @@
+= - \int_{A}   \overline{\nabla_{\perp} p}^{z}  \, dA.
+$$ (gradp-h)
+
+#### Stress Term
+
+Likewise, the stress tensor integrated over the surface may be converted to a volume integral with Gauss' theorem ([Kundu et al. 2016](https://www.amazon.com/dp/012405935X/) p. 125 eqn 4.20b),
+
+$$
+\int_{\partial V(t)} \boldsymbol{\tau} \cdot \mathbf{n} \, dA
+= \int_{V(t)} \nabla_{3D} \cdot \boldsymbol{\tau} \, dV
+$$ (stress-Gauss)
+
+for clarity, this can be written in index notation as
+
+$$
+\int_{\partial V(t)} n_i \tau_{ij} \, dA
+= \int_{V(t)} \frac{\partial}{\partial x_i} \left( \tau_{ij} \right) \, dV
+$$ (stress-Gauss-index)
+
+Taking only the horizontal ($j$=1,2),
+
+$$
+ \int_{V(t)} \frac{\partial}{\partial x_i} \left( \tau_{ij} \right) \, dV
+= \int_{A} \int_{z^{\text{bot}}}^{z^{\text{top}}} 
+ \frac{\partial}{\partial x_i} \left( \tau_{ij} \right) 
+\, dz \, dA
+=  \int_{A} 
+  \overline{
+ \frac{\partial}{\partial x_i} \left( \tau_{ij} \right) }^z dA
+$$ (gradp-h)
+
+#### Horizontal momentum
+
+Putting the pressure and stress term into [](h-momentum), and using Gauss' Theorem on the advection, the horizontal momentum equation is
+
+$$
+& \frac{d}{dt} \int_{A} \int_{z^{\text{bot}}}^{z^{\text{top}}} \rho \, {\bf u} \, dz \, dA
++
+   \int_{A}\nabla_\perp \cdot \left( \int_{z^{\text{bot}}}^{z^{\text{top}}}\rho\, {\bf u} \otimes {\bf u} \, dz \right) dA
+ + \int_{A}\left[ \rho\, {\bf u} \left(w - w_r \right) \right]_{z=z^{\text{top}}} \, dA
+ - \int_{A}\left[ \rho\, {\bf u} \left(w - w_r \right) \right]_{z=z^{\text{bot}}} \, dA
+\\ & \; =
+- \int_{A} \int_{z^{\text{bot}}}^{z^{\text{top}}} \rho \,  \mathbf{f} \times \mathbf{u} \, dz \, dA
+- \int_{A} \int_{z^{\text{bot}}}^{z^{\text{top}}} \rho \, \nabla_\perp \Phi \, dz \, dA
+ - \int_{A} \int_{z^{\text{bot}}}^{z^{\text{top}}} 
+  \nabla_{\perp} p \, dz \, dA
++ \int_{A} \int_{z^{\text{bot}}}^{z^{\text{top}}} 
+ \frac{\partial}{\partial x_i} \left( \tau_{ij} \right) 
+\, dz \, dA.
+$$ (h-momentum-p-tau)
+
+Dividing by $\rho_0$ and taking vertical averages using [](#def-thickness) and [](#def-pseudo-thickness),
+
+$$
+\frac{d}{dt} \int_{A} \tilde{h}\, \overline{  {\bf u} }^{\tilde z} \, dA
++
+   \int_{A}\nabla_\perp \cdot \left( \tilde{h}\, \overline{{\bf u} \otimes {\bf u} }^{\tilde z} \right) dA
+& + \int_{A}\left[ {\bf u}\, \tilde{w}_{tr} \right]_{z=z^{\text{top}}} \, dA
+ - \int_{A}\left[ {\bf u}\, \tilde{w}_{tr} \right]_{z=z^{\text{bot}}} \, dA
+\\ & \; =
+- \int_{A} \tilde{h} \,\overline{  \mathbf{f} \times \mathbf{u} }^{\tilde z} \, dA
+- \int_{A} \tilde{h} \,\overline{  \nabla_\perp \Phi }^{\tilde z} \, dA
+- \int_{A} \frac{1}{\rho_0} \overline{\nabla_{\perp} p}^{z}  \, dA
++ \int_{A} \frac{1}{\rho_0} \overline{ \frac{\partial}{\partial x_i} \left( \tau_{ij} \right) }^z dA
+$$ (h-momentum-p-tau)
+
+Taking the limit as $A \rightarrow 0$, we arrive at the local, horizontally continuous form:
+
+$$
+\frac{\partial \tilde{h}\, \overline{  {\bf u} }^{\tilde z} }{\partial t} 
```

Comment:
I think we need to clarify why $d/dt$ becomes $\partial/\partial t$ here.

Reply:
@xylar I'm confused by the d/dt vs \partial / \partial t throughout, especially when D/Dt is introduced later for the material derivative.  Should we really have ordinary derivatives here?  I feel like I'm missing an important, obvious point.

Reply:
My understanding (though this isn't explicitly stated) is that $d/dt$ follows the control volume (which can move in the vertical), $\partial/\partial t$ is in the Eulerian frame and $D/Dt$ follows a fluid parcel.

Reply:
Thanks @xylar I appreciate that clarification.  I'll fix this in the version I'm working on as I confused the two forms (reynolds transport and eulerian)